dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -34,6 +34,7 @@ Future<TaskResult> runWebBenchmark(WebBenchmarkOptions benchmarkOptions) async {
     await flutter('clean');
     await evalFlutter('build', options: <String>[
       'web',
+      '--no-tree-shake-icons', // local engine builds are frequently out of sync with the Dart Kernel version
```

Comment:
...meaning tree shaking icons doesn't work? Or causes crashes?

Reply:
It causes crashes. This issue will be gone when we do the monorepo, because then we'd be building everything from sources and so everything should sync up.

Reply:
These crashes are indeed annoying (and to me, seemingly random)